src/assets/languages/catalan/sentences.txt

```diff
@@ -0,0 +1,261 @@
+Les sardanes tenen un ordre bàsic i en aquest sentit són un plaer.
+El plaer ordenat és la voluptuositat.
+Una sardana són quinze minuts justos de voluptuositat.
+No dic de sensualitat.
+Dic exactament el contrari: de voluptuositat.
+En el meu temps les sardanes s'arrossegaven utilitzant un ritme en l'oscil·lació de les espatlles ple de sonsònia, picant els temps amb una certa sorna.
+Ara les sardanes se salten, sobre un espai de terra molt petit, limitat.
+El cos oscil·la poc.
+Es manté dret, vertical.
+Llavors, els braços s'alçaven només fins a mig aire i hom donava un sacsejament a les espatlles una mica esfotrassat.
+Ara els braços s'alcen més drets, més alts, amb més vigor, i les espatlles es mantenen més rígides.
+Aquestes dues maneres tan distintes de ballar, quan són presentades amb exageració, cauen en la caricatura i són horripilants; conservades en el seu punt just, tenen les seves excel·lències.
+Les sardanes s'han de ballar amb naturalitat.La cobla Montgrins (1927) Aquest és el principi —l'immortal principi— vàlid en la manera moderna i en l'antiga.
+En la manera antiga, la ritmació es produïa sobre un fris horitzontal; en la moderna, la ritmació és més vertical, sobre un espai reduït.
+El resultat és el mateix: el ritme penetra fins al moll dels ossos, hom va perdent pes i es torna lleuger i la sardana és un impuls cap a una comptabilitat inconscient i entusiasta, cap a un neguit, ordenat, de moviment.
+En l'època de què estic parlant, quan les sardanes estaven reduïdes a l'àmbit del meu país, al vessant pirinenc del Rosselló, amb una projecció sobre les dolces terres de la Selva, el seu sabor era purament local.
+Semblaven racons de malenconia subtil i greu de la sensibilitat empordanesa, en allò que aquesta sensibilitat pugui tenir de més contemplatiu, de més embadalit en el nostre fi, antic, somniós paisatge, en les nostres ruïnes.
+Però a mesura que aquesta música sortí de l'ou inicial i es vessà per l'àrea catalana, rebé alguna sotragada que modificà el seu esperit.
+La sardana es tornà més vibrant, més granada, d'un color més arravatat i lluent.
+Abandonà l'ombra trista dels xiprers del meu país i aparegué en altres ambients, davant d'altres paisatges, en medis més complexos i exigents.
+Aquest primer procés va de Ventura a Garreta, passant pel vell senyor Serra, de Peralada, si la memòria no m'és infidel.
+Per a comprendre aquest procés pot ésser de gran utilitat una frase que es troba en les biografies de Pep Ventura.
+En les evocacions, els biògrafs fan constar, recollint una tradició viva, que quan aquest, amb la tenora als llavis, tocava una sardana, «semblava que s'hi adormia...».
```

Comment:
Are « and » characters typeable on catalan keyboards? If no maybe replace with regular quotation marks

Reply:
Ha! good catch!! Yes, it should be regular quotation marks :)